<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ludo</class>
 <widget class="QMainWindow" name="Ludo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ludo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="Board" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>375</width>
        <height>375</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>375</width>
        <height>375</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="Dice" name="dice">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>75</height>
           </size>
          </property>
          <property name="icon">
           <iconset resource="Ludo.qrc">
            <normaloff>:/dices/face1</normaloff>:/dices/face1</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>60</width>
            <height>60</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
          <property name="type" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>35</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Status" name="player1" native="true">
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="type" stdset="0">
         <number>1</number>
        </property>
        <property name="turn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Status" name="player2" native="true">
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="type" stdset="0">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Status" name="player3" native="true">
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="type" stdset="0">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Status" name="player4" native="true">
        <property name="minimumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>215</width>
          <height>35</height>
         </size>
        </property>
        <property name="type" stdset="0">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>626</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mood</string>
    </property>
    <addaction name="actionMode2P"/>
    <addaction name="actionMode3P"/>
    <addaction name="actionMode4P"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionMode2P">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 Player</string>
   </property>
  </action>
  <action name="actionMode3P">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 Player</string>
   </property>
  </action>
  <action name="actionMode4P">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>4 Player</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Board</class>
   <extends>QWidget</extends>
   <header location="global">Board.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Status</class>
   <extends>QWidget</extends>
   <header location="global">Status.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Dice</class>
   <extends>QPushButton</extends>
   <header location="global">Dice.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Ludo.qrc"/>
 </resources>
 <connections/>
</ui>
